### ISSUE TITLE
Dart Explorer

### ISSUE THREAD

**Issue by radicaled** [radicaled/dart-tools#11]
After having used dart-tools for a bit of LD30, I found myself switching back and forth from Dart Editor.

The main reason was unfamiliar APIs: I really wanted to explore the classes available from the packages I was using. In Dart Editor, I did it by declaring a variable and doing a "go to definition" on it. I wasn't using code completion so much as a quick typing tool, but as a way to explore class structure and see the arguments / return types.

To alleviate this need, I think dart-tools should have a 'Dart Explorer': a pane with a search engine that lets you quickly interrogate the analysis server to see what kind of types are available and what their arguments and return values are.

Basically, a sort of an interactive "dartdoc" inside of Atom that covers all known types instead of a particular package.
